photutils/aperture/regions2aperture.py

```diff
@@ -0,0 +1,111 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This module defines tools to convert regions.Region to Aperture.
+"""
+
+import astropy.units as u
+
+from photutils.aperture import (CircularAnnulus, CircularAperture,
+                                EllipticalAnnulus, EllipticalAperture,
+                                RectangularAnnulus, RectangularAperture,
+                                SkyCircularAnnulus, SkyCircularAperture,
+                                SkyEllipticalAnnulus, SkyEllipticalAperture,
+                                SkyRectangularAnnulus, SkyRectangularAperture)
+
+
```

Comment:
```suggestion
__all__ = ['region_to_aperture']


```

This is needed to be part of the public API in the Sphinx docs.